buildpackrunner/runner.go

```diff
@@ -45,47 +53,59 @@ func newDescriptiveError(err error, message string, args ...interface{}) error {
 	return descriptiveError{message: fmt.Sprintf(message, args...), err: err}
 }
 
-type Release struct {
-	DefaultProcessTypes buildpackapplifecycle.ProcessTypes `yaml:"default_process_types"`
-}
-
 func New(config *buildpackapplifecycle.LifecycleBuilderConfig) *Runner {
 	return &Runner{
 		config: config,
 	}
 }
 
-func (runner *Runner) Run() error {
+func (runner *Runner) Run() (string, error) {
 	//set up the world
 	err := runner.makeDirectories()
 	if err != nil {
-		return newDescriptiveError(err, "Failed to set up filesystem when generating droplet")
+		return "", newDescriptiveError(err, "Failed to set up filesystem when generating droplet")
 	}
 
 	err = runner.downloadBuildpacks()
 	if err != nil {
-		return err
+		return "", err
 	}
 
-	//detect, compile, release
-	detectedBuildpack, detectedBuildpackDir, detectOutput, ok := runner.detect()
-	if !ok {
-		return newDescriptiveError(nil, buildpackapplifecycle.DetectFailMsg)
+	err = runner.cleanCacheDir()
```

Comment:
We do a `runner.config.IsMultiBuildpack` in `cleanCacheDir`. Can we move this function call a couple of lines down and remove the internal check?